include/measurement_kit/report/entry.hpp

```diff
@@ -26,6 +26,13 @@ class Entry : private nlohmann::json {
             return JsonDomainError();
```

Comment:
On second thought, users would probably use the `Entry` to parse the result. So it would probably make sense to make `Entry` easier to use, as I've tried to do here.